.github/workflows/hive-tests.yml

```diff
@@ -87,4 +98,4 @@ jobs:
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
           ./mvnw install ${MAVEN_FAST_INSTALL} -am -pl :presto-hive
       - name: Run Hive Dockerized Tests
-        run: ./mvnw test ${MAVEN_TEST} -pl :presto-hive -P test-hive-insert-overwrite
\ No newline at end of file
+        run: ./mvnw test ${MAVEN_TEST} -pl :presto-hive -P test-hive-insert-overwrite
```

Comment:
There doesn't seem a change in these lines. Please fix the diff.

Reply:
Removed. Generally it is expected to have a new line at the end of a file.